mappings/net/minecraft/client/gui/WrittenBookScreen.mapping

```diff
@@ -21,8 +21,11 @@ CLASS czz net/minecraft/client/gui/WrittenBookScreen
 	FIELD f cachedPageIndex I
 	FIELD g lastPageButton Ldaw;
 	FIELD h nextPageButton Ldaw;
+	FIELD i playPageTurnSound Z
```

Comment:
What about just `pageTurnSound`?

Reply:
Sure. That one and the couple in `ContainerScreen` were somewhat difficult to name.